proto/alliance/alliance.proto

```diff
@@ -23,34 +36,36 @@ message AllianceAsset {
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
     (gogoproto.nullable)   = false
   ];
+  // set a bound of weight range to limit how much reward weights can scale. 
+  RewardWeightRange reward_weight_range = 3 [(gogoproto.nullable) = false];
```

Comment:
@phamminh0811 instead of adding `reward_weight_range` here as index 3. Protobuf convention would be to do append-only and add the new variable at the bottom with index 10.